test/acceptance/tests.py

```diff
@@ -1101,6 +1112,94 @@ def test_train_self_peer_staff(self, peer_grades_me):
                 [(u"YOUR SELF ASSESSMENT", u"Good"), (u"YOUR SELF ASSESSMENT", u"Excellent")],
             ])
 
+@ddt.ddt
+class FeedbackOnlyTest(OpenAssessmentTest, FullWorkflowMixin):
+    """
+    Test for a problem that only accepts feedback. Will make and verify peer, self, and staff assessments.
+    """
+    def setUp(self):
+        super(FeedbackOnlyTest, self).setUp("feedback_only", staff=True)
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    def generate_feedback(self, assessment_type, feedback_type):
+        return "{}: {} feedback".format(assessment_type, feedback_type)
+
+    def assess_feedback(self, self_or_peer=""):
+        if self_or_peer != "self" and self_or_peer != "peer":
+            raise AssertionError("assert_feedback only works for self or peer assessments")
+        page = self.self_asmnt_page if self_or_peer == "self" else self.peer_asmnt_page
+        page.wait_for_page()
+        page.submit_assessment()
+
+    @retry()
+    @attr('acceptance')
+    def test_feedback_only(self):
+        # Make submission
+        user, pwd = self.do_submission()
+
+        # Make self assessment
+        self.self_asmnt_page.visit()
+        self.self_asmnt_page.wait_for_page()
+        self.self_asmnt_page.provide_criterion_feedback(self.generate_feedback("self", "criterion"))
+        self.self_asmnt_page.provide_overall_feedback(self.generate_feedback("self", "overall"))
+        self.self_asmnt_page.assess("self", [0])
+        self.self_asmnt_page.wait_for_complete()
+        self.assertTrue(self.self_asmnt_page.is_complete)
+
+        # Staff assess all available submissions
+        self.do_staff_assessment(
+            options_selected = [0],
```

Comment:
`options_selected` takes a list of indices, such that the `i-th` element represents the option to choose for criterion `i`. Since we only have one scored criterion, and we want to choose the first (zero-indexed) option for that criterion, the list becomes `[0]`. I'll add a comment to this effect.